README.md

```diff
@@ -5,4 +5,78 @@
 [![license](https://badgen.net/github/license/zitadel/saml/)](https://github.com/zitadel/saml/blob/master/LICENSE)
 [![release](https://badgen.net/github/release/zitadel/saml/stable)](https://github.com/zitadel/saml/releases)
 [![Go Report Card](https://goreportcard.com/badge/github.com/zitadel/saml)](https://goreportcard.com/report/github.com/zitadel/saml)
-[![codecov](https://codecov.io/gh/zitadel/saml/branch/master/graph/badge.svg)](https://codecov.io/gh/zitadel/saml)
\ No newline at end of file
+[![codecov](https://codecov.io/gh/zitadel/saml/branch/master/graph/badge.svg)](https://codecov.io/gh/zitadel/saml)
+
+## What is it
+
+This project is a server implementation for the "SAML" (Security Assertion Markup Language) standard written for `Go`.
+
+For signing and signature verification other already existing implementations like `github.com/russellhaering/goxmldsig`
+and `github.com/amdonov/xmlsig`.
+
+## Basic Overview
+
+The most important packages of the library:
+<pre>
+/pkg
+    /provider definitions and implementation of a SAML provider (Identity provider)
+        /serviceprovider definitions and implementation of a SAML user (Service provider)
+        /xml definitions of SAML xml messages
+        /checker helper to abstract the SAML standard in the processes
+        /signature implementation to handle and create SAML signature
+</pre>
+
+## Features
+
+|                   | POST-binding | Redirect-binding | Artifact-binding | Request signing | Response signing | Metadata signing | Response encryption | Assertion Query/Request | Attribute Query | NameID Mapping |
+|-------------------|--------------|------------------|------------------|-----------------|------------------|---------|--------------------|-------------------------|-----------------|----------------|
+| Identity provider | yes          | yes              | no               | yes             | yes              | yes              | no                | no                      | yes             | no             |
```

Comment:
```suggestion

Supported SAML features:

| Feature | Identity provider |
| --- | --- | 
| POST-binding | yes |
| Redirect-binding | yes |
| Artifact-binding | [no](https://github.com/zitadel/zitadel/issues/3089) |
| Request signing | yes |
| Response signing | yes |
| Metadata signing | yes |
| Response encryption | [no](https://github.com/zitadel/zitadel/issues/3090) |
| Assertion Query/Request | no |
| Attribute Query | yes |
| NameID Mapping | no |
```

Reply:
Just a suggestion. Tabulating it top-bottom makes it imo easier to read. 
Also added the links to existing issues for reference.

Reply:
looks also better to me, just wanted to do it in the same way it is in the oidc repo